src/js/utils/can-autoplay.js

```diff
@@ -56,11 +56,18 @@ export const AUTOPLAY_ENABLED = 'autoplayEnabled';
 export const AUTOPLAY_MUTED = 'autoplayMuted';
 export const AUTOPLAY_DISABLED = 'autoplayDisabled';
 
+let autoplayPagePromises = {};
+
 export function canAutoplay (mediaPool, { cancelable, muted = false, allowMuted = false, timeout = 250 }) {
     const element = mediaPool.getTestElement();
+    const key = `${muted}-${allowMuted}`;
```

Comment:
We can eliminate one more test case, as there are really one three possibilities:
1. `muted` is `false` - `allowMuted` is `true`,
2. `muted` is `false` - `allowMuted` is `false`,
3. `muted` is `true` - `allowMuted` is irrelevant,

```js
const key = muted ? `${muted}` : `${muted}-${allowMuted}`;
const key = muted ? 'muted' : `${allowMuted}`; // Even shorter if we want to.
```